team-validator.js

```diff
@@ -18,17 +18,18 @@ class Validator {
 		this.tools = Tools.mod(this.format);
 	}
 
-	validateTeam(team) {
+	validateTeam(team, hideNicknames) {
 		let format = Tools.getFormat(this.format);
-		if (format.validateTeam) return format.validateTeam.call(this, team);
-		return this.baseValidateTeam(team);
+		if (format.validateTeam) return format.validateTeam.call(this, team, hideNicknames);
+		return this.baseValidateTeam(team, hideNicknames);
 	}
 
-	prepTeam(team) {
-		return PM.send(this.format.id, team);
+	prepTeam(team, locked) {
```

Comment:
You have this variable named `locked`, `hideNicknames`, `checkNicknames`...

I think you should just consistently name it something like `removeNicknames`